resources/assets/components/blocks/ActionStatsBlock/ActionStatsLeaderboard.js

```diff
@@ -51,20 +52,45 @@ const ActionStatsLeaderboard = ({ actionId }) => {
         <div className="p-6 bg-white lg:w-2/3">
           {leaders.map(leader => {
             const { impact, school, location, id } = leader;
-
+            let circleBgColor;
             rank += 1;
 
+            switch (rank) {
+              case 1:
+                circleBgColor = 'bg-yellow-500';
+                break;
+              case 2:
+                circleBgColor = 'bg-purple-500';
+                break;
+              case 3:
+                circleBgColor = 'bg-blurple-500';
+                break;
+              default:
+                circleBgColor = 'bg-blurple-500';
+            }
+
             return (
               <div
-                className="mx-auto md:flex md:items-center md:justify-between border-b border-gray-300 border-solid md:pb-4 mb-4"
+                className={classnames(
+                  'mx-auto md:flex md:items-center md:justify-between',
+                  {
+                    'border-b border-gray-300 border-solid md:pb-4 mb-4':
+                      rank !== 3,
```

Comment:
This is minor, but this is another spot where it took me a minute to think about what was going on. Using `<` I think could help indicate that we'll add this for all rows except the last.

We could DRY defining the `3` here and in the GraphQL query, either as a const within our component function, but also potentially as an optional `count` prop that defaults to 3.
```suggestion
                      rank < count,
```